diff --git a/node_modules/vocs/_lib/app/components/Outline.js b/node_modules/vocs/_lib/app/components/Outline.js
index ad7b9a4..cdb7523 100644
--- a/node_modules/vocs/_lib/app/components/Outline.js
+++ b/node_modules/vocs/_lib/app/components/Outline.js
@@ -1,6 +1,6 @@
 import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { Fragment, useEffect, useMemo, useRef, useState } from 'react';
-import { useLocation } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
 import { useConfig } from '../hooks/useConfig.js';
 import { useLayout } from '../hooks/useLayout.js';
 import { debounce } from '../utils/debounce.js';
@@ -152,7 +152,7 @@ function Items({ activeId, items, levelItems, onClickItem, setActiveId, }) {
                 }
                 return nextItems;
             })();
-            return (_jsxs(Fragment, { children: [_jsx("li", { className: styles.item, children: _jsx("a", { "data-active": isActive, href: `${pathname}${hash}`, onClick: () => {
+            return (_jsxs(Fragment, { children: [_jsx("li", { className: styles.item, children: _jsx(Link, { "data-active": isActive, to: `${pathname}${hash}`, onClick: () => {
                                 onClickItem?.();
                                 setActiveId(id);
                             }, className: styles.link, children: text }) }), nextLevelItems && (_jsx(Items, { activeId: activeId, levelItems: nextLevelItems, items: items, onClickItem: onClickItem, setActiveId: setActiveId }))] }, id));
